### PR TITLE
Be able to shut up activity stream

### PR DESCRIPTION
**Pull request by relrod**
This approach is ~loosely~ inspired by what AWX does.

### CODE REVIEW

**Comment by relrod** [comment]
~well...~

~That didn't go so well.~

Okay, just going to copy the AWX approach here entirely.

File: ansible_base/activitystream/signals.py [hunk 1]
```diff
@@ -1,11 +1,36 @@
...
+    def __bool__(self):
+        return self.enabled
+
+
+activitystream_enabled = ActivityStreamEnabled()
+
+
+@contextmanager
+def no_activity_stream():
+    activitystream_enabled.enabled = False
```
Comment (AlanCoding) [inline ansible_base/activitystream/signals.py:24]: I do kind of like that AWX maintained a `prior_value` store here. It's not that you'd ever switch True-False-True-False, but you would logically want a True-False-False nesting

```python
with no_activity_stream():
  with no_activity_stream():
    pass
  obj.save()
```

In this case, I would still want `obj.save()` to leave no activity stream entry. This is perfectly testable and all.

**Review by AlanCoding — APPROVED** [review]
My comment is more of a suggestion since I don't think it's _actually_ production-relevant.

**Comment by relrod** [comment]
Good suggestion, I'll add that back in.

**Pull request merged**